zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/MultiTenancyValidator.java

```diff
@@ -71,7 +70,7 @@ public static Optional<ProblemDetail> validateAuthorization(
    */
   public static Either<ProblemDetail, String> validateTenantId(
       final String tenantId, final boolean multiTenancyEnabled, final String commandName) {
-    final var hasTenantId = !StringUtils.isBlank(tenantId);
+    final var hasTenantId = tenantId != null && !tenantId.trim().isEmpty();
```

Comment:
🙃 Isn't there an `isBlank` nowadays in the String type since Java 11, so we can write `tenantId != null && !tenantId.isBlank()`?

Reply:
Who are you, so wise in the ways of Strings

Reply:
I think someone mentioned it in one of my PRs a couple of months back, I'm just sharing the TIL experience 😄